requirements/edx/edx-private.txt

```diff
@@ -34,5 +34,5 @@ git+https://github.com/open-craft/problem-builder.git@v2.0.4#egg=xblock-problem-
 ubcpi-xblock==0.5.3
 
 # Vector Drawing and ActiveTable XBlocks (Davidson)
--e git+https://github.com/open-craft/xblock-vectordraw.git@ded76fc8f6c99ba4949ed57a7bf62a1f12e44683#egg=xblock-vectordraw
+-e git+https://github.com/open-craft/xblock-vectordraw.git@v0.2#egg=xblock-vectordraw==0.2
```

Comment:
Before this change, non-editable installs actually didn't work (I fixed the issue here: https://github.com/open-craft/xblock-vectordraw/pull/9/files). So for future updates, we should be able to remove the editable flag. :)